meson_options.txt

```diff
@@ -21,6 +21,47 @@ option('extra_include_dirs', type : 'array', value: [],
 option('extra_lib_dirs', type : 'array', value: [],
   description: 'non-default directories to be searched for libs')
 
+option('enable-tap-tests', type : 'boolean', value : false,
```

Comment:
Let's have this default to on for now...

Reply:
Done.